inventory-service/src/test/java/com/petclinic/inventoryservice/datalayer/Product/ProductRepositoryTest.java

```diff
@@ -150,6 +150,55 @@ public void testFindProductByProductId() {
                 })
                 .verifyComplete();
     }
+    @Test
+    public void shouldFindProductByInventoryIdAndOneCharNameRegex() {
+
+        String regexPattern = "(?i)^n.*";
```

Comment:
I really like the way you used the regex expression in your testing. It really shows how versatile you are in your craft Matthew keep it up and great work.

Reply:
Wow thank you Alexei Timbro, I am very grateful to hear such words gracefully come out of this comment. I will keep up the work and I hope you do too 😉.